staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -2807,6 +2943,17 @@ func withRuleMessageAndMessageExpression(s schema.Structural, rule, message, mes
 	return s
 }
 
+func withReasonAndFldPath(s schema.Structural, rule, fieldPath string, reason *metav1.StatusReason) schema.Structural {
```

Comment:
nit: Don't abbreviate words in function names. Just use `withReasonAndFieldPath`